Source/Anchorage.swift

```diff
@@ -537,7 +553,7 @@ public struct EdgeGroup {
     public var trailing: NSLayoutConstraint
 
     public var horizontal: [NSLayoutConstraint] {
-        return [leading, trailing].flatMap { $0 }
+        return [leading, trailing]
```

Comment:
In an earlier version, the leading/trailing/etc properties were optional, so the flatmap was required. This is no longer the case, however, so it's good to remove the unneeded maps